h2o-py/h2o/h2o.py

```diff
@@ -555,7 +557,7 @@ def import_sql_select(connection_url, select_query, username, password, optimize
 
 
 def parse_setup(raw_frames, destination_frame=None, header=0, separator=None, column_names=None,
-                column_types=None, na_strings=None, skipped_columns=None):
+                column_types=None, na_strings=None, skipped_columns=None, custom_non_data_line_markers = None):
```

Comment:
does not match formatting of previous args

Reply:
Resolved.